src/breeze/actions/emails/show.cr

```diff
@@ -0,0 +1,38 @@
+abstract class Breeze::Emails::Show < BreezeAction
+  param plain : Bool = false
+
+  get "/emails/:email_name" do
```

Comment:
This is just rendering the raw email. The downside here is that we don't get access to see the other email fields like subject, to, cc, etc... Another option would be to make this action just render the normal Breeze layout, and have the page use an iframe to handle this logic here. Then outside of the frame, we could load the other info. That would require having a new action which makes a secondary call to load the email. This could also just be an update we handle later... This implementation at least gives us more than we have currently.

Reply:
Since most emails will have to render their own doctype, html, etc... I'd have to render in an iframe. I guess I'll just do that in this one so I can add in the subject and all that stuff.